mobile_verifier/src/reward_shares.rs

```diff
@@ -42,14 +157,11 @@ impl RadioShares {
 }
 
 #[derive(Default)]
-pub struct OwnerShares {
+pub struct RewardShares {
```

Comment:
if we're renaming this should we be more specific and call it `PocShares` as distinct from `TransferShares` but both still falling under the umbrella of `RewardShares` since they're both rewards?